omiclearn/__init__.py

```diff
@@ -14,9 +14,9 @@
 ]
 __python_version__ = ">=3.7,<3.10"
 __classifiers__ = [
-    #"Development Status :: 1 - Planning",
+    # "Development Status :: 1 - Planning",
     # "Development Status :: 2 - Pre-Alpha",
-     "Development Status :: 3 - Alpha",
+    "Development Status :: 3 - Alpha",
     # "Development Status :: 4 - Beta",
     # "Development Status :: 5 - Production/Stable",
```

Comment:
We could probably move to Prod at some point.

Reply:
Thanks for that! I added to the list.